content/blog/posts/2022-07-08-go-serverless-functions-with-ember-and-sst-on-aws.md

```diff
@@ -0,0 +1,200 @@
+---
+authorId: jaredgalanis
+categories:
+  - ember.js
+  - sst
+  - javascript
+  - go
+  - serverless
+date: '2022-07-12'
+slug: go-serverless-functions-with-ember-and-sst-on-aws
+title: Go Serverless Functions with Ember and SST on AWS
+---
+
+You may recall in our [last post](https://shipshape.io/blog/serverless-functions-ember-glimmer/) on this subject that we made use of some tooling provided by the platforms offered by Vercel and Netlify to introduce serverless functions into Ember and Glimmer applications.
```

Comment:
- A bit confused with this sentence/would suggest a less wordy version - is the platform Vercel/Netlify or Ember/Glimmer, or something else? Suggestions based on me reading platforms as Ember/Glimmer 
- hyperlinking a more descriptive selection of text is helpful for seo

You may recall in our [last post on serverless functions](https://shipshape.io/blog/serverless-functions-ember-glimmer/) that we made use of ~some~ the tooling introduced by Vercel and Netlify for Ember and Glimmer applications.